CRM/Export/BAO/Export.php

```diff
@@ -1528,12 +1526,11 @@ public static function getExportStructureArrays($returnProperties, $processor) {
     $metadata = $headerRows = $outputColumns = $sqlColumns = array();
     $phoneTypes = CRM_Core_PseudoConstant::get('CRM_Core_DAO_Phone', 'phone_type_id');
     $imProviders = CRM_Core_PseudoConstant::get('CRM_Core_DAO_IM', 'provider_id');
```

Comment:
Are these variables still used anywhere?

Reply:
Yeah - in this function they still are - hope to change that later on